src/pages/workspace/accounting/netsuite/export/NetSuiteExportConfigurationPage.tsx

```diff
@@ -0,0 +1,258 @@
+                config?.nonreimbursableExpensesExportDestination !== CONST.NETSUITE_EXPORT_DESTINATION.EXPENSE_REPORT,
+        },
+        {
+            type: 'toggle',
+            title: translate('workspace.netsuite.exportToNextOpenPeriod'),
+            isActive: !!config?.exportToNextOpenPeriod,
+            switchAccessibilityLabel: translate('workspace.netsuite.exportToNextOpenPeriod'),
+            onCloseError: () => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.EXPORT_TO_NEXT_OPEN_PERIOD),
+            onToggle: () => Connections.updateNetSuiteExportToNextOpenPeriod(policyID, !config?.exportToNextOpenPeriod, config?.exportToNextOpenPeriod ?? false),
+            pendingAction: config?.pendingFields?.exportToNextOpenPeriod,
+            errors: ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.EXPORT_TO_NEXT_OPEN_PERIOD),
+        },
+    ];
+
+    return (
+        <ConnectionLayout
+            displayName={NetSuiteExportConfigurationPage.displayName}
+            headerTitle="workspace.accounting.export"
+            headerSubtitle={config?.subsidiary ?? ''}
+            title="workspace.netsuite.exportDescription"
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN]}
+            policyID={policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            contentContainerStyle={styles.pb2}
+            titleStyle={styles.ph5}
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.NETSUITE}
+        >
+            {menuItems
+                .filter((item) => !item.shouldHide)
+                .map((item) => {
+                    if (item.type === 'divider') {
+                        return <View style={styles.dividerLine} />;
+                    }
+                    if (item.type === 'toggle') {
+                        const {type, shouldHide, ...rest} = item;
+                        return (
+                            <ToggleSettingOptionRow
+                                key={rest.title}
+                                // eslint-disable-next-line react/jsx-props-no-spreading
+                                {...rest}
+                                wrapperStyle={[styles.mv3, styles.ph5]}
+                            />
+                        );
+                    }
+                    return (
+                        <OfflineWithFeedback
+                            key={item.description}
+                            pendingAction={item.pendingAction}
+                            errors={item.errors}
+                            errorRowStyles={[styles.ph5]}
+                            onClose={item.onCloseError}
+                        >
+                            <MenuItemWithTopDescription
+                                title={item.title}
+                                interactive={item?.interactive ?? true}
```

Comment:
This is by default true because it's not set then let's remove `item?.interactive` check?